lib/similarity_utils.py

```diff
@@ -21,3 +22,70 @@ def iou(lista1: List[Any], lista2: List[Any]) -> float:
         intersection = set(lista1).intersection(set(lista2))
         union = set(lista1).union(set(lista2))
         return len(intersection) / len(union)
+
+
+def sort_two_lists(list1: List[Any], list2: List[Any], reverse: bool = True):
+    """metoda sortuje dwie listy równocześnie
+    list1 jest sortowana po wartościach, a kolejność elementów w list2 zależy od sortowania list1
+    reverse = True oznacza kolejność malejącą, a reverse = False oznacza kolejność rosnącą"""
+    zipped_lists = zip(list1, list2)
+    list1, list2 = zip(*sorted(zipped_lists, reverse=reverse))
+    return list1, list2
```

Comment:
Co w przypadku typowania outputu takiej funkcji? W przypadku gdy piszę:
`def sort_two_lists(list1: List[Any], list2: List[Any], reverse: bool = True) -> tuple[List[Any], List[Any]]:`
to przy importowaniu/wykonywaniu metody w notatniku wyskakuje mi błąd `TypeError: 'type' object is not subscriptable`.

Reply:
Bo powinieneś napisać `Tuple` zamiast `tuple`

Reply:
Zrobione